editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -337,6 +337,22 @@ interface ComponentProps {
   nodeId: Store.nodeId;
 }
 
+function List(props: ComponentProps) {
+  const totalUnits = props.passport.data?.[`${props.node.data.fn}.total.units`];
+  const totalListItems =
+    props.passport.data?.[`${props.node.data.fn}.total.listItems`];
+  const summary = totalUnits
+    ? `${totalUnits} units total`
+    : `${totalListItems} items total`;
+
+  return (
+    <>
+      <Box component="dt">{props.node.data.title}</Box>
```

Comment:
this could also be schema "type", but I think node title is most consistent with other components and gives Editors more control here since schemas will be hard-coded for initial launch ?

Reply:
Agreed, I think this is the best approach, until proven otherwise